Heart/DataBaseSpace/databasenode.h

```diff
@@ -110,29 +110,29 @@ class HEARTSHARED_EXPORT DataBaseNode : public AbstractNode
      * @brief changeTrust This implementation of change trust is change trust node or user by self id.
      * All changes of trust saving into local database.
      * @param id This is id of user of other network member object.
-     * @param diff This is dfference of trust.
+     * @param diff This is difference of trust.
      * @return true if trust of user changed successful.
      */
     virtual bool changeTrust(const QVariant &id, int diff);
 
     /**
-     * @brief sendData This method is some as AbstractNode::sendData but try send data to the id.
+     * @brief sendData This method is some as AbstractNode::sendData but it try send data to the id.
      *  This implementation prepare object to sending.
      * @param resp This is sending object to the nodeId.
      * @param nodeId This is id of target node.
      * @param req This is header of request.
-     * @return true if data sendet seccussful
+     * @return true if data sender successful.
```

Comment:
this i meant 'send'

This method return true if data sender successful.

```suggestion
     * @return true if a data send successful.
```